internal/store/peering_connections.go

```diff
@@ -157,12 +167,13 @@ func (s *Store) AzureContainers(projectID string) ([]atlasv2.CloudProviderContai
 func (s *Store) AWSContainers(projectID string) ([]atlasv2.CloudProviderContainer, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		result, _, err := s.clientv2.NetworkPeeringApi.ListPeeringContainerByCloudProvider(s.ctx, projectID).
-			PageNum(0).
-			ItemsPerPage(maxPerPage).
-			ProviderName("AWS").
-			Execute()
-
+		params := &atlasv2.ListPeeringContainerByCloudProviderApiParams{
+			GroupId:      projectID,
+			ProviderName: pointer.Get("AWS"),
+			ItemsPerPage: pointer.Get(maxPerPage),
+			PageNum:      pointer.Get(0),
```

Comment:
can you confirm if this command has pagination and add it if not?

Reply:
sorry, I don't understand your comment

Reply:
seems like this store is using some defaults to the pagination that should be coming from the command, I'm asking if we can fix that and have the command handle pagination

Reply:
same, only projectID as param